charts/avatar/templates/avatar-api.yaml

```diff
@@ -45,8 +45,8 @@ spec:
               containerPort: 8000
               protocol: TCP
           resources:
-            {{- toYaml .Values.resources | nindent 12 }}
-      {{- with .Values.nodeSelector }}
+            {{- toYaml .Values.worker.resources | nindent 12 }}    
```

Comment:
this is the api deployment, but you're referencing .Values.worker here. I don't think you meant to do that did you?

Reply:
that was a typo , cleaned